test/org/sagebionetworks/bridge/dynamodb/DynamoStudyTest.java

```diff
@@ -79,6 +81,9 @@ public void studyFullySerializesForCaching() throws Exception {
         study = BridgeObjectMapper.get().readValue(json, DynamoStudy.class);
         assertNull(study.getStormpathHref());
         assertEquals("Study", node.get("type").asText());
+
+        // Deserialize back to a POJO and verify.
+        DynamoStudy deserStudy = BridgeObjectMapper.get().readValue(json, DynamoStudy.class);
```

Comment:
If we're going to have Task.create() we should follow through with things like Study.create() and .readValue(json, Study.class), but that's a separate PR.